<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncreaseDecreaseDialog</class>
 <widget class="QWidget" name="IncreaseDecreaseDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_param">
       <property name="text">
        <string>Parameter:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Operand</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_constant">
          <property name="text">
           <string>Constant</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QSpinBox" name="spin_constant">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QRadioButton" name="radio_variable">
          <property name="text">
           <string>Variable</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_variable"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="combo"/>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="combo_param">
       <item>
        <property name="text">
         <string>MaxHP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MaxSP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DEX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AGI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>INT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Operation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_increase">
          <property name="text">
           <string>Increase</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radio_decrease">
          <property name="text">
           <string>Decrease</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="check_battle_knockout">
       <property name="text">
        <string>Allow Knockout in Battle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo</tabstop>
  <tabstop>combo_param</tabstop>
  <tabstop>radio_increase</tabstop>
  <tabstop>radio_decrease</tabstop>
  <tabstop>radio_constant</tabstop>
  <tabstop>spin_constant</tabstop>
  <tabstop>radio_variable</tabstop>
  <tabstop>combo_variable</tabstop>
  <tabstop>check_battle_knockout</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_constant</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_constant</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_variable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
